server/app/files.go

```diff
@@ -350,6 +350,7 @@ func (a *App) CopyAndUpdateCardFiles(boardID, userID string, blocks []*model.Blo
 	newFileNames, err := a.CopyCardFiles(boardID, blocks, asTemplate)
 	if err != nil {
 		a.logger.Error("Could not copy files while duplicating board", mlog.String("BoardID", boardID), mlog.Err(err))
+		return err
```

Comment:
This alters the flow of control from what happens right now. I'm not sure whats the reason we don't return, but if this is not strictly needed for the scope of this PR, I'd skip returning from here and avoid any unknown issue. We currently lack unit tests in this repo so its not the easiest thing to explore and validate this change.